### ISSUE TITLE
configure.php replaces instances of `Alley` with author name that should remain as `Alley`

### ISSUE THREAD

**Issue by mogmarsh** [alleyinteractive/create-wordpress-project#58]
### Description of the bug

When I run `php configure.php`, some strings that should stay as `Alley` are changed to the author name (`Greg Marshall`) if I accept all the default prompts.
This leads to issues such as 
`use Greg Marshall\WP\Types\Feature;` instead of use Alley\WP\Types\Feature;`
and
```
This project is actively maintained by [Greg Marshall
Interactive](https://github.com/alleyinteractive). Like what you see? [Come work
with us](https://alley.co/careers/).

- [Greg Marshall](https://github.com/Greg Marshall)
- [All Contributors](../../contributors)
```

### Steps To Reproduce

run `php configure.php` enter an author name other than `Alley`
Check the changed files in the plugin for instances of the author name you provided.

**Comment by kevinfodness** [comment]
Related to https://github.com/alleyinteractive/create-wordpress-plugin/issues/298

**Comment by anubisthejackle** [comment]
I believe this was closed with #105. Closing this issue and if its still happening after that update we can re-open.
